p2p/pre2p/telemetry/metrics.go

```diff
@@ -0,0 +1,20 @@
+package pre2p_telemetry
+
+const (
+	// How to define metrics in this file?
+	//
+	// Must define metrics using following template: MetricName_MetricType_Attribute
+	// e.g: type= timeseries, name=nodes_alive_counter, attribute=name
+	// will result in: NODES_COUNT_COUNTER_TIME_SERIES_NAME="nodes_alive_counter"
```

Comment:
Given that you describe the template above as `MetricName_MetricType_Attribute`, should this not be `NodesAliveCounter_Timeseries_Name`?

Reply:
I meant that template for the variable name, and not the actual metric. The value of the metric name should just be semantic in my opinion. Wdyt @Olshansk